File: PE1/PE1/Problem6.fs
namespace Program

module Problem6 =

    let SumOfSquares =
        [1..100]
        |>Seq.map(fun x -> x*x)
        |>Seq.sum

    let SquareOfSums =
        [1..100]
        |>Seq.sum
        |>fun x -> x * x

    let Solve =
        SquareOfSums - SumOfSquares

File: ProjectEuler/Problems/Problem6.cs
using System;
using System.Linq;

namespace ProjectEuler.Problems
{
    public class Problem6 : Problem
    {
        protected override long DoSolutionSolve()
        {
            var sumOfSquares = Enumerable.Range(1, 100).Select(number => number * number).Sum();
            var squareOfSum = Math.Pow(Enumerable.Range(1, 100).Sum(), 2);

            Progress = 100;
            return (int)squareOfSum - sumOfSquares;
        }
    }
}
